.github/workflows/publish-stable.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
 
       - name: npm publish
-        run: pnpm release-plan publish
+        run: pnpm release-plan publish --publish-branch=stable
```

Comment:
don' we want `--publish-branch=stable|main`, since we merge `stable` into `main` frequently?

Reply:
once we sync this it won't be a problem any more. For example https://github.com/embroider-build/embroider/pull/1721 didn't even try to update the yaml branches

also it's better to be explicit with this. 

and also we didn't test if that syntax works (if the pnpm docs says it does that's fine but I'm extra cautious around releasing things). We know what this PR has does work because I've done it manually twice.

So in summary, no i don't think we should change this 😂

Reply:
ah ok, I see [what you mean](https://github.com/embroider-build/embroider/pull/1724/files/fbffd7a761762f6da8683dfb235e2acac457d7ee#diff-ee4f3ba7c21547934a6f659fce2ab1c2a42251b904c6844fad629c9f28a991aeR11)

Legit